generator.yaml

```
@@ -195,6 +195,8 @@ resources:
         - MissingParameter
         - ValidationError
     hooks:
+      delta_pre_compare:
+        code: preCompareUpgradePolicy(a, b)
```

Just in case we need more logic in the future...
```suggestion
        code: customPreCompare(a, b)
```